hack/install-etcd.sh

```diff
@@ -23,7 +27,7 @@ fi
 
 # setup a private GOPATH so the build can succeed
 export GOPATH="${PWD}/gopath"
-rm -f "${GOPATH}/src/github.com/coreos/etcd"
+rm -rf "${GOPATH}/src/github.com/coreos/etcd"
```

Comment:
Be careful: https://github.com/openshift/origin/commit/0c804382069f5592727a853015729d34a9f5bf8c